app/_gateway_entities/plugin.md

```diff
@@ -133,8 +133,74 @@ This can be [adjusted dynamically](#dynamic-plugin-ordering) using the plugin's
 You can override the [priority](#plugin-priority) for any {{site.base_gateway}} plugin using each plugin’s `ordering` configuration parameter. 
 This determines plugin ordering during the `access` phase, and lets you create dynamic dependencies between plugins.
```

Comment:
I'd like a link to a page about custom logic/the different phases like: https://docs.konghq.com/gateway/latest/plugin-development/custom-logic/